oval/util.go

```diff
@@ -448,7 +449,7 @@ func lessThan(family, newVer string, packInOVAL ovalmodels.Package) (bool, error
 	}
 }
 
-var rhelDownStreamOSVerPattern = regexp.MustCompile(`\.[es]l(\d+)(?:_\d+)?(?:\.(centos|rocky|alma))?`)
+var rhelDownStreamOSVerPattern = regexp.MustCompile(`\.[es]l(\d+)(?:_\d+)?(?:\.(centos|rocky|alma|fedora))?`)
```

Comment:
Fedora seems to include `fc` in the release, not `el`. (The epel package is supposed to be different).
```console
[root@54c54d7d7d5d /]# rpm -qa
libgcc-11.2.1-1.fc35.x86_64
...
```